runelite-client/src/main/java/net/runelite/client/plugins/worldhopper/WorldHopperPlugin.java

```diff
@@ -560,9 +561,18 @@ private void hop(boolean previous)
 
 			world = worlds.get(worldIdx);
 
-			// Check world region if filter is enabled
-			if (config.quickHopRegionFilter() != RegionFilterMode.NONE && world.getRegion() != config.quickHopRegionFilter().getRegion())
+			// Check if quick-hop is disabled.
+			if (!config.quickHopAURegion() && !config.quickHopDERegion()
+					&& !config.quickHopGBRegion() && !config.quickHopUSRegion())
```

Comment:
Same as with the added configs, condition continuations should be indented by only one additional level.
```suggestion
				&& !config.quickHopGBRegion() && !config.quickHopUSRegion())
```